plugins/other/pacman

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+# DESCRIPTION
+#
+# This plugin monitors the number of pending updates in pacman.
+#
+# WARNINGS
+#
+# This plugin will issue a warning state whenever any package can be updated.
+#
+# ENVIRONMENT
+#
+# env.WARNING The number of updates at which a warning will be issued. If not
+#             set, no warnings are issued.
+#
+# MAGIC MARKERS
+#
+# #%# family=contrib
+# #%# capabilities=autoconf
+
+case $1 in
+	config)
+		cat <<'EOM'
+graph_args --base 1000 -l 0
+graph_title Pending updates
+graph_vlabel updates
+graph_draw no
+updates.label updates
+updates.info Current number of pending updates
+EOM
+		if [[ -n $WARNING ]]; then
+			echo updates.warning $WARNING
+		fi
+		;;
+
+	autoconf)
+		hash checkupdates &> /dev/null && echo yes || echo no
```

Comment:
In a weird world, I could imagine, that:
* someone tries to use this plugin on a non-arch system
* somewhen `pacman` switches to another provider of the `checkupdates` functionality (with a different name)
Thus I would recommend to tell the user why this plugin fails.

Regarding `hash`: thank you for sharing this. As a dash/sh user, I will try to remember `command -v` from now on.